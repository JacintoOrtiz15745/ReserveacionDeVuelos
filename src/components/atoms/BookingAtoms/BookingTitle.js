import React from 'react';
import {Text} from 'react-native';
import {styles} from '../../../styles/BookingStyles';

const BookingTitle = (props) => {
  return <Text style={styles.FlightTitle} testID='title'>{props.title}</Text>;
};

export default BookingTitle;
