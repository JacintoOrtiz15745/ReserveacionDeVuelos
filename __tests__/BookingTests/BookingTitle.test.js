import React from 'react';
import {shallow} from 'enzyme';
import BookingTitle from '../../src/components/atoms/BookingAtoms/BookingTitle';

describe('component renders correctly', () => {
  const props = {
    title: 'some text',
  };
  const wrapper = shallow(<BookingTitle {...props} />);
  test('snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
  test('text matches with prop', () => {});
  const title = wrapper.findWhere((node) => node.prop('testID') === 'title');
  expect(title.props().children).toEqual(props.title);
});
