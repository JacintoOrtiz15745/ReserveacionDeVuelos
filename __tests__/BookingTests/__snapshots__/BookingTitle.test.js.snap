// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component renders correctly snapshot 1`] = `
<Text
  style={
    Object {
      "color": "black",
      "fontSize": 35,
      "fontWeight": "bold",
      "marginBottom": 80,
      "width": 300,
    }
  }
  testID="title"
>
  some text
</Text>
`;
